client/layout/guided-tours/docs/TUTORIAL.md

```diff
@@ -57,19 +57,18 @@ Now we're ready to actually start writing our tour.
 First we'll need to create a directory tour, under `tours`. In there, we create two files: `meta.js` and `index.js`.
 `meta.js` contains the metadata for a tour. Here's an empty boilerplate:
 
-```JavaScript
+```javascript
```

Comment:
I wonder if we should be consistent and just use ```js everywhere.

Reply:
It is not the same, If/when https://github.com/leo-buneev/eslint-plugin-md/pull/8 is merged, we will be able to enable https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-fenced-code-flag that can limit the list of valid languages.

Reply:
TIL 👍